lada/fellow/forms.py

```diff
@@ -4,6 +4,8 @@
 
 from lada.models import Fellow
 
+# rodo_cyrograf = 'Wyrażam zgodę na przetwarzanie moich danych osobowych w zakresie ……………………… w celu ……………………………………, zgodnie z Rozporządzeniem Parlamentu Europejskiego i Rady (UE) 2016/679 z dnia 27 kwietnia 2016 r. (ogólne rozporządzenie o ochronie danych) oraz zgodnie z klauzulą informacyjną dołączoną do mojej zgody.'
+rodo_cyrograf = 'Wyrażam zgodę na przetwarzanie moich danych osobowych, zgodnie z Rozporządzeniem Parlamentu Europejskiego i Rady (UE) 2016/679 z dnia 27 kwietnia 2016 r. (ogólne rozporządzenie o ochronie danych) oraz zgodnie z klauzulą informacyjną dołączoną do mojej zgody.'
```

Comment:
Nie jestem specem od RODO, ale mnie uczyli, że żeby klauzula informacyjna była legitna (tj. zmieniała coś w stosunku do strony bez klauzuli) musi zawierać m. in.:

- cel przetwarzania informacji
- wyznaczenie administratora danych osobowych
- kontakt do wnoszenia skarg (zwykle do IODO) 
- informację komu dane mogą zostać przekazane (lub że nie będą przekazane nikomu)

Bez tego klauzula RODO kompletnie nic nie zmienia i nadal działa się na zasadach ogólnych, że można wykorzystywać jedynie dane esencjonalne do funkcjonowania organizacji, czy jakoś tak.

Reply:
ta wykomentowana jest ze strony z iod, tylko trzeba ją uzupełnić, dlatego wrzuciłem ją tylko w komentarz póki co